frontend/src/pages/modelServing/ModelServingContext.tsx

```diff
@@ -77,7 +86,18 @@ const ModelServingContextProvider: React.FC = () => {
     dataConnectionRefresh();
   }, [servingRuntimeRefresh, inferenceServiceRefresh, dataConnectionRefresh]);
 
+  const {
+    kServe: { installed: kServeInstalled },
+    modelMesh: { installed: modelMeshInstalled },
+  } = useServingPlatformStatuses();
+
+  const notInstalledError =
```

Comment:
Elevate the check here to the context layer only for the installation of both platforms, now it will show the following page when both platforms are not installed.

<img width="1725" alt="Screenshot 2023-11-16 at 6 28 13 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/379693bd-4b99-4258-ac2e-43bb852894be">